Sources/Public/DotLottie/DotLottieFileHelpers.swift

```diff
@@ -268,4 +217,71 @@ extension DotLottieFile {
     }
   }
 
+  /// Loads an DotLottie from a specific filepath synchronously.
+  /// - Parameter filepath: The absolute filepath of the lottie to load. EG "/User/Me/starAnimation.lottie"
+  /// - Parameter dotLottieCache: A cache for holding loaded lotties. Defaults to `LRUDotLottieCache.sharedCache`. Optional.
+  public static func loadedFrom(
```

Comment:
I could also imagine being ok with merging this change if we give this method a more scary sounding name like:

```suggestion
  public static func loadSynchronouslyBlockingCurrentThread(
```

with documentation explaining that this performs file system operations like decompressing a zip archive.

What do you think?